app/views/activities/_newModal.html.slim

```diff
@@ -0,0 +1,15 @@
+<div id="newModal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```

Comment:
You mean, we should add the contents of `_newModal.html.slim` to `_form.html.slim` ?
And maybe give it a better name, right? As we'll have a "Edit Form" soon, too.